scripts/cipublish.d/00-upload-ingest-jar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${RF_DEBUG}" ]]; then
+    set -x
+fi
+
+DIR="$(dirname "$0")"
+
+function upload_ingest_jar {
+
+    if [[ -z "${1}" ]]; then
+        echo "This function requires a hash to uniquely
+              identify the upload jar"
+        exit 1
+    else
+        echo "Using upload hash: ${1}"
+        UPLOAD_HASH="${1}"
+    fi
+
+    pushd "${DIR}/.."
+    echo "Uploading Spark Ingest Jar to S3"
+    aws s3 cp \
+        "app-backend/ingest/target/scala-2.11/rf-ingest.jar" \
+        "s3://rasterfoundry-jars/ingest/rf-ingest-${UPLOAD_HASH}.jar"
```

Comment:
The only issue I was unsure about is if we wanted to have one bucket for all artifacts or namespace by environment. I leaned towards a single bucket for all jars/artifacts with the idea that it would encourage re-use, but I can also see the case for namespacing it to limit blast-radius.